pkg/ee/default-application-catalog/applicationdefinitions/argocd-app.yaml

```diff
@@ -41,6 +41,13 @@ spec:
             chartVersion: 6.0.0
             url: https://argoproj.github.io/argo-helm
       version: v2.10.0
+    - template:
+        source:
+          helm:
+            chartName: argo-cd
+            chartVersion: 7.7.17
+            url: https://argoproj.github.io/argo-helm
+      version: v2.13.3
```

Comment:
Newer version is available, which fixes few CVEs

https://github.com/argoproj/argo-cd/releases/tag/v2.13.4

Reply:
Please update update the chartVersion to the appropriate version which is most likely https://github.com/argoproj/argo-helm/blob/56c5a31c9e4367f643f8458fe8d8c18674c50f0f/charts/argo-cd/Chart.yaml#L2-L6